runtime/src/tests/misc.rs

```diff
@@ -103,3 +103,19 @@ fn submit_proof_weights_composition_should_ignore_aggregate_if_no_domain() {
         assert!(proof_size < <Runtime as pallet_aggregate::Config>::WeightInfo::on_proof_verified().proof_size()/2);
     });
 }
+
+#[test]
+fn check_version() {
+    let v = std::env!("CARGO_PKG_VERSION");
+    let v_num = v
+        .split('.')
+        .map(|x| x.parse::<u32>().unwrap())
+        .enumerate()
+        .fold(0, |a, (step, dec)| a + dec * 1000_u32.pow(step as u32));
```

Comment:
Shouldn't we reverse before `enumerate()`?
Otherwise, for example, a crate version `0.14.1` would result in `v_num == 1014000`, while it should be `14001`.